src/main/resources/standardTestCases/bad_docker_name.test

```diff
@@ -0,0 +1,7 @@
+name: bad_docker_name
+testFormat: workflowfailure
+
+files {
+  wdl: bad_docker_name/bad_docker_name.wdl
+  inputs: hello/hello.inputs
+}
```

Comment:
I didn't follow this conversation, but I'd agree with (what I think was) Ruchi's point - that adding a metadata block like this to the end of the `.test` file would be good:
```
metadata {
  failures: [ "bad docker name" ]
}
```